include/ftl/task_scheduler.h

```diff
@@ -83,9 +87,9 @@ class TaskScheduler {
 	std::atomic<EmptyQueueBehavior> m_emptyQueueBehavior;
 
 	enum class FiberDestination {
-		None = 0,
```

Comment:
Can we make enum classes UpperCamelCase?

Reply:
Sure.

Reply:
Note that that enum classes and plain enums are handled identically within Resharper